pkg/cli/status/status.go

```diff
@@ -0,0 +1,98 @@
+package status
+
+import (
+	"context"
+	"regexp"
+	"strings"
+
+	"github.com/google/go-github/v45/github"
```

Comment:
```suggestion
	"github.com/google/go-github/v47/github"
```

Reply:
ah the editor auto completion, still can't take it off after removing every ref to it in code, running make vendor will make it comes back to go.mod 🤷🏻